test/cmdlineargs.jl

```diff
@@ -240,6 +240,10 @@ let exename = `$(Base.julia_cmd()) --startup-file=no --color=no`
         @test startswith(read(`$exename --help`, String), header)
     end
 
+    # Test to make sure that command line --help and --help-hidden do not return a description which is more than 100 characters wide
+    @test isempty(filter(x->length(x) > 100, readlines(`$exename -h`)))
```

Comment:
Minor suggestion to save allocations (not that this is a performance-critical code, but why not):
```suggestion
    @test all(x -> length(x) ≤ 100, eachline(`$exename -h`))
```

Reply:
The `isempty` test will actually print the lines that were too long on failure wheras the `all` test will not: https://github.com/JuliaLang/julia/pull/58835#discussion_r2173268637